misc/github_wiki/creates_markdown.py

```diff
@@ -0,0 +1,86 @@
+"""Creates markdown github wiki from bibtex files."""
```

Comment:
This is only for the citations page, right? Maybe it might better to give the file a more specific name, so that we don't have a clash if we create other types of pages in the future